wdn/templates_4.1/scripts/script-font.js

```diff
@@ -0,0 +1,25 @@
+define([
+	'jquery',
+	'css!js-css/script-font'
+], function($, WDN, require) {
+    var initd = false;
+
+    return {
+        initialize: function() {
+            // protect against multiple initializations
+            if (initd) {
+                return;
+            }
+            initd = true;
+
+            // Load Sanelma script font
+            var $scriptFont = $("<link>", {
+                "rel" : "stylesheet",
+                "type" : "text/css",
+                "href" : "js-css/script-font.css"
```

Comment:
I admit the difference to the other font plugins is subtle, but this is a different beast.
https://github.com/unl/wdntemplates/blob/72bcf10bf2cffa31a43c182d337d905d4b572725/wdn/templates_4.1/scripts/smallcaps.js#L19 represents an _external_ (vendor) CSS file that needs to be loaded into the head. https://github.com/unl/wdntemplates/blob/72bcf10bf2cffa31a43c182d337d905d4b572725/wdn/templates_4.1/scripts/smallcaps.js#L3 represents a _local_ (built) CSS file that will automatically be injected into the page when the script module is loaded.

They are indeed different files that serve a similar purpose. In this case, the local file contains an `@import` of the external file. So the need for a head appending link may not be necessary. I need to check to see how requirejs builds this CSS dependency to ensure it handles the import and relative font urls.